lib/util/package-rules/index.spec.ts

```diff
@@ -167,9 +167,9 @@ describe('util/package-rules', () => {
       ],
     };
     const res = applyPackageRules(dep);
-    expect(res.automerge).toBeTrue();
-    const res2 = applyPackageRules({ ...dep, depName: 'anything' });
-    expect(res2.automerge).toBeFalse();
+    expect(res.automerge).toBeFalse();
+    const res2 = applyPackageRules({ ...dep, depName: 'foo' });
+    expect(res2.automerge).toBeTrue();
```

Comment:
why is this test changed? we shouldn't need to change any test in this complex refactor.

Reply:
I think this is a bug/false test, based on the conversation on slack with @rarkins 

If no match is applied it should default to match all. In the original test case that would mean both results would be false as the package rule would apply to both.

Reply:
We should extract this fix to a separate PR, just to be sure